domains/service.go

```diff
@@ -176,3 +188,15 @@ func (svc service) ListDomains(ctx context.Context, session authn.Session, p Pag
 	}
 	return dp, nil
 }
+
+func validateID(id string) error {
```

Comment:
@felixgateru  
Lets move this validation to `api/http/common.go` and rename the function to `ValidateName`
For Now lets use ValidateUUID in transport layer for ID 

So the plan for now is we provide two option for ID, 
- We provide UUID 
- Bring your own UUID 

We will introduce another field called name/alias where user can put any string with the constraint mentioned in the ticket https://github.com/absmach/supermq/issues/2689#issue-2841428635